crawl-ref/source/dat/descript/de/backgrounds.txt

```diff
@@ -11,19 +11,19 @@ an Luftmagie-Zaubern die nötige Flexibilität, um zu überleben.
 %%%%
 Arcane Marksman
 
-Arkane Scharfschützen sind für den Fernkampf bewaffnet und verwenden Magie, um ihre
+Arkane Schützen sind für den Fernkampf bewaffnet und verwenden Magie, um ihre
 Feinde anfälliger für derartige Angriffe zu machen.
 %%%%
 Artificer
 
-Artificer beginnen mit zwei Zauberstäben, einer magischen Schachfigur und einer
-Keule. Sie müssen bessere Waffen und Rüstungen finden, bevor ihre Zauberstäbe
+Artificer beginnen mit zwei Zauberstäben, einer magischen Schachfigur und einem
+Knüppel. Sie müssen bessere Waffen und Rüstungen finden, bevor ihre Zauberstäbe
```

Comment:
If we change this here, we'll have to also change it in dat/strings/de/item*.txt